src/components/SingleOrbitImage.tsx

```diff
@@ -7,21 +9,24 @@ interface SingleOrbitImageProps {
 
 const SingleOrbitImage: React.FC<SingleOrbitImageProps> = ({
   id,
-  imageDesc
+  imageDesc,
 }) => {
   const imageName = id + imageDesc;
 
+  const isMobileDevice = useIsMobileDevice();
   const needsMobileAsset = imageDesc === "_fill" || imageDesc === "_dotted";
 
-  const { image } =
-    document.documentElement.clientWidth <= 500 && needsMobileAsset
+  const { image, loading } =
+    isMobileDevice && needsMobileAsset
       ? useImage(imageName + "_mobile")
       : useImage(imageName);
 
   return (
-    <div className={`${id} ${imageDesc}`}>
-      <img src={image} id={imageName} className={imageName} alt={imageName}/>
-    </div>
+    <ShowIf value={!loading}>
+      <div className={`${id} ${imageDesc}`}>
+        <img src={image} id={imageName} className={imageName} alt={imageName} />
+      </div>
+    </ShowIf>
```

Comment:
I realized that the problem with the broken images has to do with the page rendering before the images were loaded.
I brought in this component (Complements of something Noah made and used in capstone and now I use everywhere)
It is a very clean way of only rendering a component if the value is true.

Now it wont try to render the image until it is done loading.

I added it to both OrbitTitleGroup and SingleOrbitImage and from what I can tell it is fixed now.